app/routes/__index/dns-records/$dnsRecordId.tsx

```diff
@@ -69,8 +69,8 @@ export default function DnsRecordRoute() {
         Edit DNS Record
       </Heading>
       <Text maxW="lg" mb="3" mt="2">
-        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
-        been the industry's standard dummy text ever since the 1500s
+        Choose your unique subdomain name. Then enter Type and Name that will be mapped with your
```

Comment:
"Chose your unique subdomain Name, then enter a Type and Value that will be mapped with your domain ..."